src/impl/external_commit_helper.hpp

```diff
@@ -23,8 +23,8 @@
 
 #if REALM_PLATFORM_APPLE
 #include "impl/apple/external_commit_helper.hpp"
-#elif REALM_ANDROID || REALM_PLATFORM_NODE
-#include "impl/android/external_commit_helper.hpp"
+#elif REALM_HAVE_EPOLL
```

Comment:
One downside to this is that it requires bindings using the object store on Linux but not using object store's CMake files to explicitly define `REALM_HAVE_EPOLL`. They need to conditionally compile the .cpp file anyway so it doesn't seem like a big deal. Would love to hear other's thoughts on this though.